lib/dbservice_web/views/user_view.ex

```diff
@@ -28,4 +32,21 @@ defmodule DbserviceWeb.UserView do
       country: user.country
     }
   end
+
+  def render("optimized_user.json", %{user: user}) do
```

Comment:
this is not an optimized user. This is a user with reduced data fields. I would suggest renaming it to maybe `user_with_compact_fields.json` or something.